src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
Avoid styling the `body` tag directly. Instead, use classes to apply styles to elements. This helps to maintain a separation of concerns and prevents potential conflicts when combining multiple stylesheets or when styles need to be overridden.